resources/definitions/flashforge_dreamer_nx.def.json

```
@@ -0,0 +1,48 @@
+{
+    "version": 2,
+    "name": "Dreamer NX",
+    "inherits": "flashforge_base",
+    "metadata": {
+        "visible": true,
+        "author": "Egon",
+        "manufacturer": "Flashforge",
+        "file_formats": "text/x-gcode",
+        "platform": "FlashForge_DreamerNX.obj",
+        "platform_offset": [0, 0, 0],
+        "quality_definition": "flashforge_base",
+        "machine_extruder_trains":
+        {
+            "0": "ff_dreamernx_extruder_0"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Dreamer NX" },
+        "machine_width": {"default_value": 230},
+        "machine_height": {"default_value": 140},
+        "machine_depth": {"default_value": 150},
+        "machine_center_is_zero": {"default_value": true},
+		"machine_heated_bed": {"default_value": true},
```

Mix of tabs and spaces
```suggestion
        "machine_heated_bed": {"default_value": true},
```